install/tests/Makefile

```
@@ -33,35 +33,51 @@ help: Makefile
 	@sed -n 's/^##//p' $< | column -t -s ':' |  sed -e 's/^/ /'
 	@echo
 
+upload-gcp-cluster-creds:
+	export GKE_CREDS=$$(terraform output -json gke_user_key) && \
+	echo $$GKE_CREDS > gcp-creds
```

```suggestion
		echo $$GKE_CREDS > gcp-creds
```

Worth indenting this once more as it's a multiline command